src/cli.cr

```diff
@@ -0,0 +1,17 @@
+require "./clang"
+require "./crystal_lib"
+require "option_parser"
+output_file = ""
+OptionParser.parse! do |parser|
+  parser.banner = "Usage: crystal_lib [arguments]"
```

Comment:
Some sort of clarification that the last argument should be the input file is missing here.